ui/pages/Project/components/RnaSeqJunctionOutliersTable.jsx

```diff
@@ -11,25 +10,6 @@ import ShowGeneModal from 'shared/components/buttons/ShowGeneModal'
 import { ButtonLink } from 'shared/components/StyledComponents'
 import { getLocus } from 'shared/components/panel/variants/VariantUtils'
 
-const RNA_SEQ_SPLICE_NUM_FIELDS = ['zScore', 'pValue', 'deltaPsi']
-const RNA_SEQ_SPLICE_DETAIL_FIELDS = ['type', 'readCount', 'rareDiseaseSamplesWithJunction', 'rareDiseaseSamplesTotal']
-
-const RNA_SEQ_SPLICE_COLUMNS = [
-  ...RNA_SEQ_SPLICE_NUM_FIELDS.map(name => (
-    {
-      name,
-      content: camelcaseToTitlecase(name).replace(' ', '-'),
-      format: row => row[name].toPrecision(3),
-    }
-  )),
-  ...RNA_SEQ_SPLICE_DETAIL_FIELDS.map(name => (
-    {
-      name,
-      content: camelcaseToTitlecase(name).replace(' ', '-'),
```

Comment:
Move into utils for sharing between the RNAseq result page and the search result variants.